dev/ci/ci-hott.sh

```diff
@@ -7,4 +7,4 @@ HoTT_CI_DIR=${CI_BUILD_DIR}/HoTT
 
 git_checkout ${HoTT_CI_BRANCH} ${HoTT_CI_GITURL} ${HoTT_CI_DIR}
 
-( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && make -j ${NJOBS} )
+( cd ${HoTT_CI_DIR} && ./autogen.sh && ./configure && ./etc/coq-scripts/timing/make-pretty-timed-or-error.sh -j ${NJOBS} && make validate )
```

Comment:
`dev/tools` looks like a good place? Or maybe `coq_makefile` should be made aware of it if `COQ_TIMED=1` ?

That way we could uniformly build timings for all the CIs without even having to tweak the individual script files, just setting `COQ_TIMED` at the top-level travis file. I guess that this way the coq-bench infrastructure could also take profit from that more easily.

WDYT? OMWV! @gares?